controller/controller-board/src/app/controllers/TaskController.js

```diff
@@ -64,7 +65,20 @@
             .loadAllClusterNames()
             .then(function(result) {
                 vm.clusterStatusCode = result.status;
-                vm.namespaceClusters = result.data;
+                var namespaceSet = new Set();
+                var nameSet = new Set();
+                for (var i = 0; i < result.data.length; i ++) {
+                  var namespace = result.data[i].namespace;
+                  var name = result.data[i].name;
+                  if (!(namespace in namespaceSet)) {
+                    namespaceSet.add(namespace);
+                  }
+                  if (!(name in nameSet)) {
+                    nameSet.add(name);
+                  }
+                }
+                vm.clusterNamespaces = [...namespaceSet]
```

Comment:
Tested !

<img width="856" alt="screen shot 2017-12-11 at 11 37 37 am" src="https://user-images.githubusercontent.com/14355591/33850134-b93598e4-de67-11e7-8687-1ae9691b1225.png">

Reply:
wow, thank you!